Gulpfile.js

```diff
@@ -171,6 +170,24 @@ gulp.task("build-check-test", gulp.series(
 gulp.task("build-lint-test", gulp.task("build-check-test"));
 
 // Helper functions:
+function spawnTask(command, done, args)
+{
+    let lCP;
+    if (args !== undefined)
+    {
+        lCP = spawn(command, args, { stdio: "inherit" });
+    } 
+    else
+    {
+        lCP = spawn(command, { stdio: "inherit" });
+    }
+    
+    lCP.on("error", done);
+
+    // catch non-zero exit statii so that CI can understand when task fails
```

Comment:
is `statii` a typo? im unsure cause i dont think `static` makes sense here?

Reply:
I thought statii was the plural of status, but it turns out I was a wanker -- it's both status and statuses.